recipe/meta.yaml

```diff
@@ -13,11 +13,10 @@ source:
   git_tag: v{{ version }}
   patches:
     # xgboost patches
-    - 0001-conda-Unbundle-libxgboost.-dll-dylib-so.patch
     - 0001-Force-endian-flag-in-cross-compilation-mode.patch  # [arm64 or aarch64 or ppc64le]
-    - 0002-Use-static-cudart.patch
-    # xref: https://github.com/dmlc/xgboost/pull/9602
-    - 0003-numpy-ndarray-check.patch
+    - 0002-Enable-latest-libcxx-on-MacOS.patch  # [osx]
+    - 0003-Use-mingw-w64-path.patch
+    - 0004-Undo-dmlc-xgboost-9436.patch
```

Comment:
Could someone please double check these are all the right patches for us? Also please feel free to add/remove to bring these inline with expectations